docs/source/membership/membership.md

```diff
@@ -172,11 +172,11 @@ A local MSP folder contains the following sub-folders:
 
   **Note:** If you are using a [Hardware Security Module(HSM)](../hsm.html) for key management, this folder is empty because the private key is generated by and stored in the HSM.
 
-* **signcert:** For a peer or orderer node (or in a client's local MSP) this folder contains the node's **signing key**. This key matches cryptographically the node's identity included in **Node Identity** folder and is used to sign data --- for example to sign a transaction proposal response, as part of the endorsement phase.
+* **signcert:**  For a peer or orderer node (or in a client’s local MSP) this folder contains the node’s certificate issued by CA. This certificate contains the node's **public key**. While private key is used to digitally sign a transaction, public key is used to verify that the signature attached to the transaction is valid.
```

Comment:
See original explanation of the signature of the transaction proposal response, not just the blanket 'transaction'.

Reply:
I went through the original explanation of the signature of the transaction proposal response(https://hyperledger-fabric.readthedocs.io/en/latest/peers/peers.html?highlight=transaction proposal response#phase-1-proposal). It says "A peer endorses a proposal response by adding its digital signature, and signing the entire payload using its private key." As per my understanding(reference: Digital Signature definition at https://github.com/bitcoinbook/bitcoinbook/blob/develop/ch06.asciidoc), although digital signature uses both private and public keys, essentially in this scenario, private key is always used to sign and public key is always used to verify the signature. 
So, when the original text says "This key..and is used to sign data --- for example to sign a transaction proposal response, as part of the endorsement phase.", it seems incorrect since signcert folder contains public key and is meant to verify the signature and not sign the data.

Please let me know if my understanding is incorrect.

Reply:
You are absolutely correct, in that the signercert folder does _not_ contain the private key, it contains a certificate, corresponding to the client's identity.

However, I fear that talking about public vs private keys is a little too technical here.  It is standard cryptography, that an X.509 certificate contains a public key, as well as information about signing algorithms, as well as a issuing information.  Certificates are considered to be 'public' information and may be shared, the private key is private information, corresponding to the certificate, which is secret, and is used to prove that a message purporting to be from the certificate owner is the certificate owner.

Personally, I would simply say that the certificate in the signcert directory represents the node's identity, and that that certificate's corresponding private key can be used to generate signatures which may be verified by anyone with a copy of that certificate.

Reply:
Understood. Yes, my sentence seems too technical. Your sentence seems much better. Thanks.